spec/cbra/graph_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Cbra::Graph do
+  describe "#generate!" do
+    let(:fixture_root) do
+      File.expand_path(
+        "../fixtures",
+        File.dirname(__FILE__)
+      )
+    end
+
+    let(:root) do
+      File.expand_path(
+        "../../../cbra",
```

Comment:
Just don't go so far up that you need to step back down again. Think about how this would break if I checked this repo out to a differently named directory.